test/support/livebook_case.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Kino.LivebookCase do
+  use ExUnit.CaseTemplate
+
+  setup do
+    original_gl = Process.group_leader()
+
+    gl = start_supervised!({KinoTest.GroupLeader, self()})
+    Process.group_leader(self(), gl)
+
+    on_exit(fn ->
```

Comment:
I don’t think you need to revert because the process will exit anyway?

Reply:
Right, I did it just in case, but all regular requests are forwarded to the regular group leader, so it's definitely fine to remove this either way :)